assignments/this.js

```diff
@@ -13,14 +13,60 @@
 
 // code example for Window Binding
 
+window._ = {};
```

Comment:
to show window binding here I would just console log the this keyword. the window._ = {} will generally will just equal an empty object since you're just declaring an empty variable here.